src/elf/symver.rs

```diff
@@ -597,8 +572,6 @@ pub struct Verdaux {
 /// Auxiliary][Vernaux] entries.
 #[derive(Debug)]
 pub struct VerneedSection<'a> {
-    /// String table used to resolve version strings.
-    pub verstr: Strtab<'a>,
```

Comment:
i feel much better about this now since it doesn't commit us here to anything pub